rollastic/node.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, cluster, node_id=None):
         '''
         self.cluster = cluster
         self.node_id = node_id
+        self.badcount = 0
```

Comment:
This desperately needs a better name. What are we trying to say with this? We're trying to indicate that we've had unsuccessful or bad attempts to get info, so we should say something to that affect.

unsuccessful_attempts_to_get_node_info_count = 0